### DISCUSSION TITLE
question: extensions

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3019] [Ideas]
is it planed to add extensions like sd-webui-reactor (its a face swaper) ?

**Comment by mashb1t** [comment]
This is not how Fooocus is structured, it does feature well-integrated and tightly coupled AI features with optimized resource requirements and does not have an extension system. If you find a feature useful feel free to check the existing issues marked with the label "enhancement" (both internal and external) and if you can't find any you can open a new feature request providing more informaton.
